tests/features/sbom-explorer.feature

```diff
@@ -0,0 +1,178 @@
+Feature: SBOM Explorer - View SBOM details
+  Background:
+    Given There is ingested SBOM with Package details and affected by Vulnerabilities
+
+  Scenario: View SBOM Overview
+    When User visits SBOM details Page
+    # e.g. by selecting SBOM from Search results or from Latest SBOMS on Dashboard
+    Then SBOM name should be visible in top section
+    # `top section` refers to "upper" part visible above tabs selection
+    Then selection of tabs should be visible
+    And Info tab selector should be visible
+    And Packages tab selector should be visible
+    And Vulnerabilities tab selector should be visible
+    And Dependency Analytics Report tab should not be visible
+    # D.A.R tab not present at all, this test can be removed for future versions
+
+  Scenario: View SBOM Info (Metadata)
+    When User visits SBOM details Page
+    Then Info tab is selected # by default
+    When looking at content inside selected tab
+    Then SBOM name should be visible
+    And SBOM namespace should be visible
+    And either SPDX or CycloneDX Version should be visibile
+    And SBOM License should be visible
+    And SBOM Creation date should be visible
+    And SBOM Creator should be visible
+    And Package overview panel should be visible
+    # TODO: this Package panel needs clarification
+
+  Scenario: Downloading SBOM file
+    When User visits SBOM details Page
+    Then link to Download SBOM should be visible
```

Comment:
We can remove this "Then" statement as the "Download" button validation is covered on "Scenario: View SBOM Overview"